### DISCUSSION TITLE
There is a filter that does not apply in the Bromite custom filter I created.

### DISCUSSION THREAD

**Discussion by chereare** [bromite/bromite#1096] [Q&A]
I use DCINSIDE.com.
However, even if I use the Bromite Custom Filter, the adbox will disappear.
It appears with the checked part of the next picture.

https://i.ibb.co/sR5kN4Q/Bromite.jpg

The filter value obtained by using the UBLOCK component selector was this.

m.dcinside.com ##. ADV-Inner> .adv-group
m.dcinside.com ##. ADV-GROUPIN
m.dcinside.com ##. brick-lf> .clkhigh.grid> .Adv-group

When I wrote a custom filter, an error has not occurred and the filter works normally.
However, the ad will disappear, but the adbox will not disappear.
Why does this happen?

**Comment by csagan5** [comment]
See https://github.com/bromite/bromite/wiki/AdBlocking
